### ISSUE TITLE
R on-line courses

### ISSUE THREAD

**Issue by angelini75** [wageningen/RUsers#3]
Hi everyone,

Have you ever followed an R on-line course? If so, could you reply to this issue providing a link, a summary of the course and other relevant details?

Thanks,
@wageningen

**Comment by MirjamBreure** [comment]
I'm doing the online data science specialization offered by John Hopkins University on Coursera.org:
https://www.coursera.org/specializations/jhu-data-science

If you look up the individual courses, you can enroll without having to pay (no certificate though). So far very happy with the courses, but I'm only in the second course. First course was introduction and was quite easy, finished it in a short time. The second already takes much more time. 

Cheers,
Mirjam

**Comment by angelini75** [comment]
Thank for this information @MirjamBreure . Did you check if your garduate school recognise the credits of this course? As far as I know PE&RC does it.

**Comment by MirjamBreure** [comment]
Yes, I am in PE&RC and I can add some of the credits to my Training and Supervision Plan!

**Comment by LucSteinbuch** [comment]
Not about R directly, but to get an intuition for linear algebra, I strongly recommend the [youtube playlist "Essence of linear algebra"](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)
